.github/workflows/build_and_test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         run: npm ci
 
       - name: Build Web App
-        run: npx run action gulp -- build browser
+        run: npm run action gulp -- build browser
```

Comment:
gulp does not support positional arguments. so i wrapped it in an action so things can be consistent.